test_requirements.txt

```diff
@@ -0,0 +1,17 @@
+
+# Requirements needed for running the test suite.
+astroid==2.0.4
+coverage==4.5.1
+flake8-bugbear==18.8.0
+flake8-commas==2.0.0
+flake8-comprehensions==1.4.1
+flake8-import-order==0.18
+flake8-logging-format==0.4.0
+flake8-mutable==1.2.0
+flake8==3.5.0
+pycodestyle==2.3.1
+pylint==2.1.1
+pytest-mock==1.10.0
+pytest==3.7.2
+pytest-randomly>=1.0,<2
+testing.postgresql>=1.3.0
```

Comment:
YAAAAAS... so much prefer the disposable database approach of `testing.postgres` 👍

Reply:
me too